01-arch/00-overview.md

```diff
@@ -4,70 +4,68 @@
-to concentrate on their primary tasks --- data storage and processing, and leave
-assets management and distributed system coordination to Neo and a set of Smart
-Contracts. In such approach, Blockchain is mainly used as a trusted source of
+to focus on their primary tasks --- data storage and processing, while
+assets management and distributed system coordination are left to Neo and a set of Smart
+Contracts. Under such approach, Blockchain is mainly used as a trusted source of
 truth and coordination data.
 
 ![Architecture overview](pic/overview-sc2)
 
-In \gls{mainnet} there is a NeoFS [Native
-Contract](https://medium.com/neo-smart-economy/native-contracts-in-neo-3-0-e786100abf6e),
-taking care of user's deposits and withdrawals, network settings as well as some
-other maintenance operations like trusted nodes' key listing.
+\Gls{mainnet} contains a NeoFS [Native
+Contract](https://medium.com/neo-smart-economy/native-contracts-in-neo-3-0-e786100abf6e) concerning user's deposits and withdrawals, network settings, and some
+other maintenance operations such as trusted nodes' key listing.
 
-Besides \gls{mainnet}, NeoFS relies on Neo 3.0-based
-[sidechain](https://en.wikipedia.org/wiki/Blockchain#Types) to simplify
-accounting operations, lessen Main Net burden and reduce the overall network
-maintenance costs. NeoFS Sidechain runs Smart Contracts controlling NeoFS
-network structure, users settlements, balances and other frequently changing
+To simplify
+accounting operations, lessen Main Net burden, and reduce the overall network
+maintenance costs, NeoFS utilizes Neo 3.0-based
+[sidechain](https://en.wikipedia.org/wiki/Blockchain#Types). NeoFS Sidechain runs Smart Contracts controlling NeoFS
+network structure, users settlements, balances, and other frequently changing
 data.
 
-There are two types of NeoFS nodes: Storage nodes and Inner Ring nodes.
+There are two types of NeoFS nodes. They are Storage nodes and Inner Ring nodes.
 
-The first type is responsible for receiving data from the user and reliably
-storing it according to storage policy and providing access to the data
-according to \glspl{acl}. Storage nodes are coordinated with Smart Contracts
-from Side Chain.
+The first type is responsible for receiving data from a user, reliably
+storing it as required by the storage policy, and providing access to the data
+according to applicable \glspl{acl}. Such storage nodes are coordinated with Smart Contracts
+from the Side Chain.
 
 The second type does not store user data. Inner Ring nodes monitor the NeoFS
-network health, aggregate Storage Nodes reputation ratings and perform data
+network health, aggregate Storage Nodes reputation ratings, and perform data
 audit, giving out penalties and bounties depending on the audit results. Inner
-Ring nodes listen for both Main Net and Sidechain, providing a trusted and
+Ring nodes consider both Main Net and Sidechain, providing a trusted and
```

Comment:
Here `listen` is more of a technical term, like listening a socket.

Reply:
Alright